drive/carts/pexsplore_bbs.p8

```diff
@@ -16,7 +16,7 @@ bar_color_1=12
 bar_color_2=-4
 
 cart_dir='games'
-label_dir='labels'
+label_dir='bbs' -- TODO: currently bbs is a symlink because I want to test p8.png loading. This should not be a symlink and instead games should have p8.png files when possible since those have more information.
```

Comment:
I think this was a hack, because I wanted to test loading actual carts instead of image files.